sonar-helm-for-iac/go.work.sum

```diff
@@ -2,6 +2,10 @@ github.com/creack/pty v1.1.9 h1:uDmaGzcdjhF4i/plgjmEsriH11Y0o7RKapEf/LDaM3w=
 github.com/golang/protobuf v1.5.0 h1:LUVKkCeviFUMKqHa4tXIIij/lbhnMbP7Fn5wKdKkRh4=
 github.com/kr/pty v1.1.1 h1:VkoXIwSboBpnk99O/KFauAEILuNHv5DVFKZMBN/gUgw=
 github.com/pkg/diff v0.0.0-20210226163009-20ebb0f2a09e h1:aoZm08cpOy4WuID//EZDgcC4zIxODThtZNPirFr42+A=
+github.com/sonarsource/go/src/fmtsort v1.21.8-1 h1:67vL2r63TfFPUAX/7+95NAyshy7twzSfIxZn8yvaXIs=
+github.com/sonarsource/go/src/fmtsort v1.21.8-1/go.mod h1:82Bo3Ce/xR8MRV/Y95dEx22DsQYMb7m7PZ3CnY2PX0c=
+github.com/sonarsource/go/src/text v1.21.8-2 h1:YFGZLKmG0DUO8niT1Xh1GROBFEKeRvfv9JDN8i1oMzo=
+github.com/sonarsource/go/src/text v1.21.8-2/go.mod h1:ORstQVHMkGYSWnDb0/Yvh07inphmvbS7lvMgafG98Ik=
```

Comment:
I guess there should be one, the same version for all files?

Reply:
Changed module structure a bit, now it's a single module